schemas/1.5/dbt_project-1.5.json

```diff
@@ -207,6 +207,10 @@
       },
       "additionalProperties": false
     },
+    "invalidate_hard_deletes": {
+      "type": "boolean",
```

Comment:
This should be a `"$ref": "#/$defs/boolean_or_jinja_string"`. I don't remember offhand whether you can also have a description in the mix though, you might have to sacrifice that